docs/products/postgresql/reference/pg-metrics.rst

```diff
@@ -0,0 +1,305 @@
+      - Aiven suggest to actively monitor this value and associate it with an alert. The database will stop working correctly if it runs out of disk space.
+    * - ``Disk i/o``
+      - The number of bytes read and written per second on each of the nodes.
+      -
+    * - ``Data disk usage``
+      - The amount of disk space that is in use on the service's data disk.
+      -
+    * - ``CPU iowait``
+      - The percentage of CPU time spent waiting for the disk to become available for read and write operations
+      - Aiven suggest to create an alert that is triggered when ``iowait`` goes beyond a certain threshold for an extended time. This gives you an opportunity to respond quickly when the database starts to slow down from too many read and write operations.
+    * - ``Network``
+      - The number of inbound and outbound bytes per second for a node.
+      -
+    * - ``Network (sum of all nodes)``
+      - The same as the ``Network`` graph, but values are not grouped by service node.
+      -
+    * - ``TCP connections``
+      - The number of open TCP connections, grouped by node.
+      -
+    * - ``TCP socket state total on all nodes``
+      - The number of TCP connections across all service nodes, grouped by the TCP connection state.
+      -
+
+PostgreSQL-specific metrics
+---------------------------
+
+For most metrics, the metric name identifies the internal PostgreSQL statistics view. See the `PostgreSQL documentation <https://www.postgresql.org/docs/current/static/monitoring-stats.html>`_ for more detailed explanations of the various metric values.
+
+Metrics that are currently recorded but not shown in the default dashboard include ``postgresql.pg_stat_bgwriter`` and ``postgresql.pg_class`` metrics as a whole, as well as some individual values from other metrics.
+
+PostgreSQL Overview
+"""""""""""""""""""
+
+The metrics in the PostgreSQL overview section are grouped by logical database. In addition, some metrics are grouped by host.
+
+
+.. image:: /images/products/postgresql/metrics-dashboard-pg-overview.png
+    :alt: Aiven Grafana Dashboard for PostgreSQL database Overview Section
+
+
+.. list-table::
+    :header-rows: 1
+
+    * - Parameter Name
+      - Parameter Definition
+      - Additional Notes
+    * - ``Database size``
+      - The size of the files associated with a logical database
+      - Some potentially large files that are not included in this value. Most notably, the write-ahead log (WAL) is not included in the size of the logical databases as it is not tied to any specific logical database.
+    * - ``Connections``
+      - The number of open connections to the database
+      - Each connection puts a large burden on the PostgreSQL server and this number :doc:`should typically be fairly small even for large plans <../concepts/pg-connection-pooling>`. Use connection pooling to :doc:`reduce the number of connections <../concepts/pg-connection-pooling>` to the actual database server.
```

Comment:
These two links both seem to go to the same place and I'd expect the link to the connection pooling article to come off the words "connection pooling". I'm not exactly sure what the right thing is here, can you take another look?

Reply:
Fixed